indigo/bulk_creator.py

```diff
@@ -167,7 +163,7 @@ def create_works(self, view, table, dry_run, workflow, user):
 
         for idx, row in enumerate(rows):
             # ignore if it's blank or explicitly marked 'ignore' in the 'ignore' column
-            if row.get('ignore') or not [val for val in row.values() if val]:
+            if row.get('ignore') or not [val for val in list(row.values()) if val]:
```

Comment:
No need for `list()`, we're just iterating over the values.
```suggestion
            if row.get('ignore') or not [val for val in row.values() if val]:
```